doc/guide/install.rst

```diff
@@ -352,17 +366,18 @@ For further information, see the VSCode docs on `Python Testing <https://code.vi
 Spyder
 ^^^^^^
 
-Installing Spyder into the existing Anaconda environment for CLIMADA might fail depending on the exact versions of dependencies installed.
+Installing Spyder into the existing Conda environment for CLIMADA might fail depending on the exact versions of dependencies installed.
 Therefore, we recommend installing Spyder in a *separate* environment, and then connecting it to a kernel in the original ``climada_env``.
```

Comment:
Is this still the case? I have recently not had any problems with spyder and CLIMADA in the same environment. Do we know? Else we can just leave it.

Reply:
I have no idea 🤷 @Evelyn-M, haven't you been using Spyder?

Reply:
I refrained from touching this fragile setup in the last year or so. Hence I keep using a separate spyder-env. no idea whether it is working better now, but wouldnt know why it would

Reply:
I recently tried using the Anaconda-Navigator GUI to install Spyder and that didn't work, so I also stuck the separate Spyder environment

Reply:
@chahank Let's keep it as is for now?

Reply:
Fine for me!